IanGRap/Squish
# Sprint 1 Demo

Contributors: Ian

Goal: Create a demo scene using features made up until this point for use of initial playtesting.

Changes:

- Created a scene for playtesting called "Sprint 1" under "Scenes"/"Demo Builds" folder
- Adjusted Player prefab values for jumping and dashing
- Added SFX call in BlockCollisionDelegator.cs
- Created makeshift GameManager object with GameManager.cs for reloading the scene during play
- Changed dash button from B to right bumper

How to test: 

- Load Scene "Sprint 1"
- Plug in 2 controllers (one for each sphere character) and avoid falling blocks.
- Try pushing blocks to crush your opponent.
- If you run out of blocks are someone is squished, press R to start over

Notes:

- I wanted to change the size of the blocks but it was very complicated since I had rescale each collider on the child game objects of the blocks. Is there a way to optimize this in code so I can just change 1 value?
- This PR should be reviewed in meeting before getting approval so we can all play together and discuss.
